client/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -278,6 +280,26 @@ export class JetpackConnectNotices extends Component {
 				noticeValues.status = 'is-warning';
 				noticeValues.icon = 'notice';
 				return noticeValues;
+
+			case UNKNOWN_REMOTE_INSTALL_ERROR:
+				noticeValues.text = translate(
+					'Something went wrong. You can try again, {{manualInstall}}install Jetpack manually{{/manualInstall}} ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Invalid credentials. Try again, {{manualInstall}}install Jetpack manually{{/manualInstall}} or {{support}}contact support{{/support}} for help.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
This is a good suggestion, but I might "soften" the _existing_ string by adding the "You can" in front of "try again..".

Reply:
Totally normal to be discussing strings with a bot.

Reply:
Actually, the existing "Try again..." makes sense in the invalid credentials case, and "You can try again..." is better in the unknown problem case, so I'll leave things as they are.